From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Fri, 28 Feb 2020 10:56:07 -0800
Subject: [PATCH] Fix install location of gi/overrides file

The 'libdir' option is set to /usr/lib64, but the install path prefix
here needs to be /usr/lib, since the python 3.* site-packages tree lives
there.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 bindings/python/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bindings/python/meson.build b/bindings/python/meson.build
index ecd68ab..fa115fe 100644
--- a/bindings/python/meson.build
+++ b/bindings/python/meson.build
@@ -26,7 +26,7 @@ ret = run_command([python3, '-c', get_overridedir])
 if ret.returncode() != 0
   error('Failed to determine Python 3 pygobject overridedir')
 else
-  pygobject_override_dir = join_paths(get_option('libdir'), ret.stdout().strip())
+  pygobject_override_dir = join_paths('/usr/lib', ret.stdout().strip())
 endif
 
 install_data('gi/overrides/Modulemd.py', install_dir: pygobject_override_dir)
